cetmix_tower_yaml/tests/__init__.py

```
@@ -4,3 +4,4 @@
 from . import test_plan
 from . import test_yaml_export_wizard
 from . import test_yaml_import_wizard
+from . import test_server_log
```

Flake8 (7.2.0)

[error] '.test_server_log' imported but unused

(F401)